server/src/main/java/io/unitycatalog/server/persist/CommitRepository.java

```diff
@@ -107,4 +111,24 @@ public void updateTableMetadata(Session session, Commit commit) {
     tableInfoDAO.setColumns(ColumnInfoDAO.fromList(metadata.getSchema().getColumns()));
     session.merge(tableInfoDAO);
   }
+
+  public List<CommitDAO> getLatestCommits(UUID tableId) {
+    try (Session session = SESSION_FACTORY.openSession()) {
+      Transaction tx = session.beginTransaction();
+      try {
+        Query<CommitDAO> query =
+            session.createQuery(
+                "FROM CommitDAO WHERE tableId = :tableId ORDER BY commitVersion DESC",
+                CommitDAO.class);
+        query.setParameter("tableId", tableId);
```

Comment:
Assume, start = 0, end = 99 and MAX_NUM_COMMITS_PER_TABLE=50, what will we return? [0, 49] or [50, 99]? Is it the same as DBR?